client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
@belcherj every test case needs a slightly different state, so that solution wouldn't work.

@ryelle Not a fan of having separate files for a test unless the alternative is absolutely ridiculous.

What do you folks think about this? https://github.com/Automattic/wp-calypso/pull/15034/commits/1394a56e0b28713f5e2e656aee07c31de878cff9